mapswipe_workers/tests/unittests/fixtures/build_area/projects.json

```diff
@@ -0,0 +1,30 @@
+{
+  "archived" : false,
+  "created" : "2020-02-03T14:31:08.192136Z",
+  "createdBy" : "test manager",
+  "groupMaxSize" : 0,
+  "groupSize" : 120,
+  "image" : "",
+  "isFeatured" : false,
+  "lookFor" : "buildings",
+  "name" : "test - Malawi \ntest",
+  "progress" : 0,
+  "projectDetails" : "test",
+  "projectId" : "test_build_area",
+  "projectNumber" : "1",
+  "projectRegion" : "Malawi",
+  "projectTopic" : "test",
+  "projectType" : 1,
+  "requestingOrganisation" : "test",
+  "requiredResults" : 12537,
+  "resultCount" : 0,
+  "status" : "inactive",
+  "tileServer" : {
+    "apiKey" : "AopsdXjtTu-IwNoCTiZBtgRJ1g7yPkzAi65nXplc-eLJwZHYlAIf2yuSY_Kjg3Wn",
```

Comment:
we shouldn't list the api key here. also for other potential other projects e.g. maxar we should avoid this. maybe there is even a better way of storing this in general.